app/controllers/api/api_controller.rb

```diff
@@ -9,6 +9,7 @@ class ApiController < ActionController::API
     rescue_from ActionController::BadRequest, with: :bad_request_response
     rescue_from ActiveRecord::StatementInvalid, with: :bad_request_response
     rescue_from ArgumentError, with: :bad_request_response
+    rescue_from ActiveRecord::RecordNotUnique, with: :conflict_response
```

Comment:
do we need a new response type here will `:bad_request_response` not be suitable?

Reply:
I don't mind, any other gov projects use ```conflict_response```?

Reply:
@mattonfoot changed to ```bad_request``` in 9de72bb4